src/Sulu/Component/Content/Mapper/ContentMapperInterface.php

```diff
@@ -302,13 +302,6 @@ public function orderBefore($uuid, $beforeUuid, $userId, $webspaceKey, $language
     public function orderAt($uuid, $position, $userId, $webspaceKey, $languageCode);
 
     /**
-     * TRUE ignores mandatory in save
-     * @param bool $ignoreMandatoryFlag
-     * @return $this
-     */
-    public function setIgnoreMandatoryFlag($ignoreMandatoryFlag);
```

Comment:
@wachterjohannes what was this for? also the `setNoRenaming` flag that I removed earlier. Neither seem to be used.